src/page/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,133 @@
+import { useEffect, useState } from 'react';
+import { useParams, NavLink } from 'react-router-dom';
+import cn from 'classnames';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const PeoplePage = () => {
+  const { slug } = useParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
just FYI - if you set the default value the type of variable will be defined automatically